app.go

```diff
@@ -292,9 +294,10 @@ func executeSell(cmd command) bool {
 		return true
 	}
 
-	dollarAmount.Add(cashRemainder)
-	account.AddFunds(dollarAmount)
+	_ = cashRemainder //squash unused variable warning using blank identifier
```

Comment:
Not sure what you mean. If anything it should _decrease_ the number of characters since `cashRemainder -> _`